src/structures/Message.js

```diff
@@ -235,15 +235,17 @@ class Message extends Base {
   }
 
   /**
-   * Updates the message.
+   * Updates the message and returns the old message.
    * @param {Object} data Raw Discord message update data
+   * @returns {Message}
```

Comment:
The [typings](https://github.com/MattIPv4/discord.js/blob/limit-message-edit-history/typings/index.d.ts#L980) should be updated to reflect this

Reply:
TIL privates have typings

Reply:
(∩ ͡° ͜ʖ ͡°)⊃━☆ﾟ. * ･ ｡ﾟ,